### PR TITLE
[docs] Add examples for Blitz OIDC

### PR DESCRIPTION
**Pull request by kusaleev**
## Description

Add examples and description for Blitz OIDC

## Why do we need it, and what problem does it solve?

Extends documentation with real integration examples.

## Why do we need it in the patch release (if we do)?

No needed.

## What is the expected result?

The documentation has been updated on the website.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: user-authn
type: feature
summary: Added examples for Blitz OIDC.
impact_level: low
```

### CODE REVIEW

File: modules/150-user-authn/docs/USAGE_RU.md [hunk 1]
```diff
@@ -164,13 +172,65 @@ spec:
...
+    promptType: consent 
+    scopes:
+    - profile
+    - openid
+    userIDKey: sub
+    userNameKey: email
+  type: OIDC
+```
+
+Для того чтобы корректно отрабатывал выход из приложений (происходил отзыв токена и требовалась повторная авторизация), нужно установить `login` в значении параметра `promptType`.
```
Comment (VV-corrector) [inline modules/150-user-authn/docs/USAGE_RU.md:208]: «Для того чтобы» → «Чтобы»

File: modules/150-user-authn/docs/USAGE_RU.md [hunk 2]
```diff
@@ -164,13 +172,65 @@ spec:
...
 
-Полученные в ходе выполнения инструкции `clientID` и `clientSecret` укажите в кастомном ресурсе [DexProvider](cr.html#dexprovider).
+Для обеспечения гранулированного доступа пользователя к приложениям необходимо:
+ 
+* Добавить параметр `allowedUserGroups` в `ModuleConfig` нужного приложения.
+* Добавить группы к пользователю (наименования групп должны совпадать как на стороне Blitz, так и на стороне Deckhouse).
```
Comment (VV-corrector) [inline modules/150-user-authn/docs/USAGE_RU.md:213]:
```suggestion
* добавить параметр `allowedUserGroups` в `ModuleConfig` нужного приложения;
* добавить группы к пользователю (наименования групп должны совпадать как на стороне Blitz, так и на стороне Deckhouse).
```

**Review by kirkonru — APPROVED** [review]

**Review by Zhbert — APPROVED** [review]

**Review by nabokihms — APPROVED** [review]

**Review by nabokihms — CHANGES_REQUESTED** [review]
Could you please run `make lint-markdown-fix` to satisfy the linter?

**Comment by Zhbert** [comment]
> Could you please run `make lint-markdown-fix` to satisfy the linter?

Done.

**Pull request merged**